rotkehlchen/history/events/structures/eth2.py

```diff
@@ -35,8 +35,8 @@
     int,            # timestamp
     str | None,  # location label
     str,            # amount
-    str,            # usd value
     str,            # event_subtype
+    dict[str, Any] | None,  # extra data
```

Comment:
Adjusted the types to be correct. In the follow up PR I'm going to check if they are used or not and remove if needed